packs/github/actions/create_issue.yaml

```diff
@@ -0,0 +1,27 @@
+---
+name: create_issue
+runner_type: run-python
+description: Create a Github issue.
+enabled: true
+entry_point: create_issue.py
+parameters:
+  user:
+    type: "string"
+    description: "User / organization name."
+    required: true
+  repo:
+    type: "string"
+    description: "Repository name."
+    required: true
+  title:
+    type: "string"
+    description: Title
+    required: true
+  description:
+    type: "string"
+    description: Description
+    required: false
+  assignee:
+    type: "string"
+    description: Assignee
+    required: false
```

Comment:
For milestones and labels it seems as if one has to do additional calls to GitHub's API to get their ids. The GitHub Python library only allows to lookup milestones as ints and to list them all. Would be an expensive operation to always retrieve the list and check whether the param is among them.